js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -33,31 +34,37 @@
 	let modelLoadInfo: ModelLoadInfo | undefined = undefined;
 	let selectedInputGroup: string;
 
-	const inputSamples: WidgetInputSample[] = (model?.widgetData ?? [])
-		.sort((sample1, sample2) => (sample2.example_title ? 1 : 0) - (sample1.example_title ? 1 : 0))
-		.map((sample, idx) => ({
-			example_title: `Example ${++idx}`,
-			group: "Group 1",
-			...sample,
-		}));
+	let inputSamples: WidgetInputSample[] = [];
+	let inputGroups: { group: string; inputSamples: WidgetInputSample[] }[] = [];
+	$: {
```

Comment:
Or else the widget won't update correctly when the props change (needed for hf-ndpoints)